web/styles/dark_theme.css

```diff
@@ -1067,7 +1067,8 @@
         }
     }
 
-    .panel_user_list > .pill-container {
+    .panel_user_list > .pill-container,
+    .stream_creator_details > .display_only_pill {
```

Comment:
I cannot see any difference while hovering with and without this change. Can you share some screenshot?

Reply:
This is from Firefox 124.0.1 on Linux sway wm.

| On main | On this PR |
|--------|--------|
| ![image](https://github.com/zulip/zulip/assets/133781250/13fa90b4-b397-4f76-831c-b33695560991) | ![image](https://github.com/zulip/zulip/assets/133781250/1bd07dc7-9297-446a-9a9d-66a3ee578f54) |

Reply:
Ok. I just tested it on chrome. It is fine then.